plugins/woocommerce-blocks/assets/js/blocks/product-collection/constants.ts

```diff
@@ -64,6 +64,10 @@ export const DEFAULT_QUERY: ProductCollectionQuery = {
 	timeFrame: undefined,
 	priceRange: undefined,
 	filterable: false,
+	relatedBy: {
+		categories: true,
+		tags: true,
+	},
```

Comment:
As discussed [here](https://a8c.slack.com/archives/C05UMTA7W0P/p1730808930132549?thread_ts=1730807011.817329&cid=C05UMTA7W0P), I have decided to not include "Attributes" option in this iteration as it requires good amount of efforts and value it provides is not that high comparing to Categories and Tags.